server/src/main/java/org/elasticsearch/index/search/stats/SearchStats.java

```diff
@@ -190,7 +196,10 @@ public long getSuggestTimeInMillis() {
         }
 
         public TimeValue getSuggestTime() {
-            return new TimeValue(suggestTimeInMillis);
+            if (suggestTimeInMillis >= -1) {
```

Comment:
Wait, this shouldn't be necessary? I see tests we are passing in `-1`? Those aren't realistic though, right?

Reply:
Reverted and fixed the tests, see above.